backend/src/api/v1/dto/BaseDTO.ts

```diff
@@ -0,0 +1,18 @@
+export default class BaseDTO {
+  public id: number;
+  public name: string;
+
+  constructor(id: number, name: string) {
+    this.id = id;
+    this.name = name;
+  }
+
+  get slug(): string {
```

Comment:
Good call! Actually I wanted to a pre-existing library for that. Any suggestions?